cli/src/print.rs

```diff
@@ -160,7 +160,7 @@ impl<'a, W: Write> Print<W> {
                     self.writeln(table)?;
                 }
                 false => {
-                    self.write_header(labels.iter().map(|s| s.as_str()))?;
+                    self.write_header(labels.iter().map(std::string::String::as_str))?;
```

Comment:
would it be enough with `String::as_str` rather than `std::string::String::as_str`?